docs/source/actions.rst

```diff
@@ -616,7 +616,7 @@ benefit from any pull request attributes e.g. ``{{author}}`` and also these
 additional variables:
 
     * ``{{ check_rule_name }}`` the name of the rule that triggered this action.
-    * ``{{ check_succeed }}`` will be ``True`` if all conditions matches otherwise ``False``
```

Comment:
Not sure about that one, because it's the Python bool

Reply:
Good catch!